src/components/EventInfo/Text.jsx

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components"
+import * as styleVars from "../variable"
+
+const Text = styled.div`
+  max-width: 440px;
+  font-size: 18px;
+
+  h2 {
+    color: ${styleVars.hackWhite};
+    font-size: 36px;
+    font-weight: bold;
+    margin-top: 0;
+    margin-bottom: 40px;
+  }
+
+  p {
+    color: ${styleVars.textColor};
+    margin-top: 0;
+    margin-bottom: 32px;
+  }
+
+  @media only screen and (max-width: ${styleVars.smUp}) {
```

Comment:
Inconsistent code here, we should move to setting custom breakpoints in the variables file but let's not intermix these for the time being

Reply:
I agree, I would still recommend that we merge this code in as it is taken from the `About/` `Text.jsx` file.

I will go over the code this weekend possibly and as mentioned in the questions, attempt to genericize, and remove as many magic numbers as possible.

Reply:
Yeah, this codebase has a lot of issues in that regard.
The number of duplicated, `<Text>` and `<Containers>` etc. is something we really should cut down on, and we don't really have styling standards that are enforced